cli/wsproxy.ml

```diff
@@ -85,7 +85,9 @@ let proxy (fd : Lwt_unix.file_descr) protocol localport =
       Lwt_log.warning_f "Unknown protocol, fallback to hybi10" >>= fun () ->
       Lwt.return (wsframe, wsunframe) 
   end >>= fun (frame,unframe) ->
-  with_open_connection_fd "localhost" localport ~callback:(fun localfd ->
+  let localhost = Unix.inet_addr_of_string "localhost" in
```

Comment:
I don't think this works. The string should be the ip as far as I know, that is why it was using `gethostbyname` before.

Reply:
So "127.0.0.1" would work? The background is I want to support domain sockets in addition.

Reply:
Yes, that should work

Reply:
Indeed:
```ocaml
utop # Unix.inet_addr_of_string "localhost"
;;
Exception: Failure "inet_addr_of_string".

utop # Unix.inet_addr_of_string "127.0.0.1"
;;
- : Unix.inet_addr = <abstr>
```

Reply:
The missing bits are detecting the use of a domain socket in the protocol string that initiates this.